google-cloud-storage/lib/google-cloud-storage.rb

```diff
@@ -46,6 +46,9 @@ module Cloud
     # @param [Integer] retries Number of times to retry requests on server
     #   error. The default value is `3`. Optional.
     # @param [Integer] timeout Default timeout to use in requests. Optional.
+    # @param [Integer] open_timeout Timeout to use for open timeout. Optional.
```

Comment:
It would be useful to have more details or an example about what these different timeouts should be used for. It would also be good to include the default values.

Reply:
Ack